release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -2167,6 +2167,8 @@ After upgrading, it is recommended to disable the `NET_RAW` and `SYS_CHROOT` cap
 *Important*: If you are upgrading from an earlier release, upgrade to 4.5.16 before you upgrade to 4.6.
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1874671[*BZ#1874671*]).
 
+* The {product-title} machine-api bare metal actuator currently deletes Machine objects when the underlying BareMetalHost is deleted. This behavior does not align with other cloud provider actuators, which move a Machine object to the failed phase rather than removing it altogether if the underlying cloud provider resource is deleted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1868104[*BZ#1868104*])
```

Comment:
I'd double-check the styling on the object names, but it's usually "Machine API" in running text, and unless "BareMetalHost" is a specific object, I think it needs to be "bare metal host."
```suggestion
* The {product-title} `machine-api` bare metal actuator currently deletes Machine objects when the underlying `BareMetalHost` is deleted. This behavior does not align with other cloud provider actuators, which move a Machine object to the failed phase rather than removing it altogether if the underlying cloud provider resource is deleted. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1868104[*BZ#1868104*])
```

Reply:
`BareMetalHost` is a k8s CRD type, just like `Machine`.